web/massastation/src/mirage/server.js

```diff
@@ -177,6 +204,20 @@ function mockServer(environment = ENV.DEV) {
 
         return stores;
       });
+
+      this.get(
+        'plugin/massalabs/wallet/api/accounts',
+        (schema) => {
+          let { models: accounts } = schema.accounts.all();
+
+          return accounts;
+        },
+        { timing: 500 },
+      );
+
+      this.put('websiteUploader/prepare', (schema) => {
+        return schema.create('website');
```

Comment:
this doesn't work as expected, it return {id: 4}. But it's not a problem because we don't use the response in the code

Reply:
I think you need to specify the other parameters

Reply:
then what is the purpose of the factory?

Reply:
it works for the seed but apparently not with the create

Reply:
ok, I have added a TODO comment